src/create_tracks.hpp

```diff
@@ -257,9 +257,37 @@ void process_scores(hmm const & hmm, std::vector<double> &scores,
         if (i==0 && path[i]==0) {
             starting_position=blockStartPos-1;
             starting_count=0;
+            if (path[i+1]!=0) {
+                prob = state_probabilities[starting_count][0];
+                double log_odd = compute_log_odds(prob);
+                if (log_odd < 1) {
```

Comment:
This code fragment appears multiple times now, so it would make the code more readable and easier to maintain by moving it into a separate function like

```
uint32_t compute_bed_color(const double log_odd)
{
    if (log_odd < 1)
        return 210;
    else if (log_odd > 7)
        return 0;
    else
        return 210 - 30 * int(floor(log_odd));
}
```